README.adoc

```diff
@@ -7,7 +7,10 @@ image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/
 image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml/badge.svg["Compile Examples status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml"]
 image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml/badge.svg["Spell Check status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml"]
 
-This library allows the control and configuration of the BQ24195 PMIC used on various Arduino boards.
+This library allows the control and configuration of the BQ24195 PMIC used on various Arduino boards including:
+- All MKR boards (apart from MKR 1000, Zero, WAN 1300 and Vidor 4000)
```

Comment:
Since there are only 9 boards total, it seems it will be more clear to list the ones that do have the part rather than the list of boards that don't have it.